app/src/services/realitio.ts

```diff
@@ -82,8 +87,14 @@ class RealitioService {
     const iface = new ethers.utils.Interface(realitioAbi)
     const event = iface.parseLog(logs[0])
 
+    const question: QuestionLog = RealitioQuestionLib.populatedJSONForTemplate(
```

Comment:
Seems like you could use `defaultTemplateForType('bool')` here. Seems more readable and doesn't depend on the `'@realitio/realitio-contracts` (which, btw, it's not in the package.json dependencies, don't we have a linter rule to prevent this from happening)?

Reply:
Added the suggested change, also added an eslint rule to check import of invalid libraries